packages/hash/frontend/src/components/BlockLoader/BlockLoader.tsx

```diff
@@ -38,10 +38,10 @@ import { EntityType as ApiEntityType } from "../../graphql/apiTypes.gen";
 type BlockLoaderProps = {
   accountId: string;
   blockEntityId: string;
-  blockEntityType?: ApiEntityType;
```

Comment:
In #752 I introduced this prop without appreciating that `blockEntityId` here refers to the entity id of the _wrapping_ block entity, not the entity passed to the block. The 'thing passed to the block' in these props owns the things named as `entityX` below.